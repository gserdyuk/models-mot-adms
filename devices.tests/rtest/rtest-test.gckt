* rtest test 

*MCKT 1 2 3 4 nekv26eval W = 2e-05 L = 2e-05
Drtest 1 2  drtestmodel

.model drtestmodel d r1 1000
		
* DEF SOURCES
V1 1 0 DC 1
V2 2 0 DC 0

.PRINT DC v(1) I(V1) 
.DC V1 0 2 0.5  basic

.end
