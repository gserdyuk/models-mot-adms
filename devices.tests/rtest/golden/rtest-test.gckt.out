Gnucap 2009.12.07 RCS 26.136
The Gnu Circuit Analysis Package
Never trust any version less than 1.0
Copyright 1982-2009, Albert Davis
Gnucap comes with ABSOLUTELY NO WARRANTY
This is free software, and you are welcome
to redistribute it under the terms of 
the GNU General Public License, version 3 or later.
See the file "COPYING" for details.
D: already installed, replacing
stashing as D:0
d: already installed, replacing
stashing as d:0
* rtest test 
#           v(1)       I(V1)     
 0.         0.         0.        
 0.5        0.5       -500.E-6   
 1.         1.        -0.001     
 1.5        1.5       -0.0015    
 2.         2.        -0.002     
